blockbase/sass/blocks/_site-title.scss

```diff
@@ -0,0 +1,3 @@
+.wp-block-site-title a {
+	text-decoration: none;
```

Comment:
I was wondering if we shouldn't be able to do this via theme.json. I tested and it works, but it applies to the wrapper instead of the anchor, so not all themes will have the rule working. I think this is a GB issue. At most, blockbase should say "inherit", and let the theme decide with theme.json until GB fixes the problem:
<img width="385" alt="Screenshot 2021-10-06 at 19 14 34" src="https://user-images.githubusercontent.com/3593343/136252064-10606030-f959-4286-9a9e-a6f7055187d2.png">

with:


```
"core/site-title": {
				"typography": {
					"textDecoration": "none",
					"fontSize": "var(--wp--preset--font-size--normal)",
					"fontWeight": 700,
					"textTransform": "uppercase"
				}
			},
```